### ISSUE TITLE
spectrum.get_populations() fails to parse specair output dictionnary

### ISSUE THREAD

**Issue by gh4ag** [radis/radis#134]
### 🐛 Describe the bug
When using "primary_code="specair" the return populations dictionnary contains a first set of keys ("atoms", "molecules") that is not handled by the function get_populations of Spectrum object.

### 🖥 Steps To Reproduce
```python
from pyspecair import one_slab
s = one_slab(wavelength_min=400,
                    wavelength_max=810,
                    Tgas=2000,
                    Trot=2000,
                    Tvib=5000,
                    Telec=1e4,
                    pressure=1,
                    path_length=1,
                    mole_fractions={'N2':1},
                    isotopes={'N2':1},
                    self_absorption=True,
                    verbose=False,
                    medium='air',
                    use_cached=True,
                    primary_code='specair')
print(s.get_populations(molecule="N2"))
```
### 🎯 Expected behavior
this should return the same as 

`print(s.populations["molecules"]["N2"]) `

But only for "specair" as primary code.

### 💡 Possible solutions
Create a case for "primary_code='specair'" in def get_populations of spectrum/spectrum.py

**Comment by erwanp** [comment]
Hello @gh4ag !! 

Unfortunately this is a Specair related problem even if Radis is finally used to receive the calculated spectrum. 

Can you post the same issue in the [Specair Gitlab](https://gitlab.com/spectralfit/specair/-/issues) ?

**Comment by dxb0920** [comment]
Dear sir, When I installed the pyspecair software in the linux system, according to the installation tutorial "git clone [git@gitlab.com](mailto:git@gitlab.com):spectralfit/specair.git", an error "sign_and_send_pubkey: signing failed: agent refused operation
Permission denied (public key). fatal: Could not read from remote repository.". On the gitlab.com website, I can't find any information about pyspecair. I tried to contact the administrator of the specair software, but have not received a reply yet. How should I solve this problem? Thanks.
